cmake/VcpkgToolchain.cmake

```diff
@@ -45,7 +45,6 @@ if(_HOST_IS_WINDOWS OR EXISTS "${_VCPKG_MONO}")
 endif()
 
 list(APPEND VCPKG_OVERLAY_PORTS "${CMAKE_SOURCE_DIR}/Scripts/Ports")
-list(APPEND VCPKG_OVERLAY_TRIPLETS "${CMAKE_SOURCE_DIR}/Scripts/Triplets")
```

Comment:
CI seems to be failing due to this line being removed. I think that you intended to remove the line above it instead.